common/local_filestore.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for using the local_filestore tool."""
+
+import os
+
+from common import new_process
+
+
+def local_filestore_command(arguments, expect_zero=True):
+    """Executes a local command with |arguments| and returns the result."""
+    return new_process.execute(arguments, expect_zero=expect_zero)
+
+
+def cp(  # pylint: disable=invalid-name
+        source,
+        destination,
+        recursive=False,
+        parallel=False):  # pylint: disable=unused-argument
+    """Executes "cp" command with |cp_arguments|."""
+    # Create intermediate folders for `cp` command to behave like `gsutil.cp`.
+    for file_or_dir_path in [source, destination]:
+        dirpath = os.path.dirname(os.path.abspath(file_or_dir_path))
+        if not os.path.exists(dirpath):
+            os.makedirs(dirpath)
+
+    command = ['cp']
+    if recursive:
+        command.append('-r')
+    command.extend([source, destination])
+    return local_filestore_command(command)
+
+
+def ls(path, must_exist=True):  # pylint: disable=invalid-name
+    """Executes "ls" command for |path|. If |must_exist| is True then it can
+    raise subprocess.CalledProcessError."""
+    # Add '-l' to behave like `gsutil.ls`.
+    command = ['ls', '-l', path]
```

Comment:
I think this needs to be `-1` not `-l` (as in the number one).

Reply:
Will double check the behavior in `gsutil.ls` later.

Reply:
Just change it to `-1` now please.

Reply:
OK. Done.